template/content/about.phtml

```diff
@@ -1,3 +1,4 @@
+<?php require_once __DIR__ . '/../../config/variables_about.php'; ?>

```

Comment:
@ReginaBorisova 
Please change this code to dynamically load variables required for a page based on a page and have in in the variables file

Reply:
Done